rpc/client/httpclient.go

```diff
@@ -249,23 +249,46 @@ func (c *HTTP) Validators(height *int64) (*ctypes.ResultValidators, error) {
 
 /** websocket event stuff here... **/
 
+type subscription struct {
```

Comment:
Why can't we use the pubsub.Subscription?

Reply:
I've removed this in https://github.com/tendermint/tendermint/pull/3269. It should've been a single PR probably